src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return (a) => {
+    return (b) => {
+      ++counter;
```

Comment:
this operator isn't obvious. Very cool that you know it and know how to use it and let's try to improve the code readability